daml-lf/kv-support/src/main/scala/com/daml/lf/kv/archives/ArchiveConversions.scala

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.kv.archives
+
+import com.daml.lf.archive.ArchiveParser
+
+object ArchiveConversions {
+
+  @throws[com.daml.lf.archive.Error]
+  def extractHash(rawArchive: RawArchive): String =
```

Comment:
should we keep the naming consistent with other functions in `//daml-lf` and start them with `assert`?